omod/src/main/webapp/resources/scripts/services/privilegeService.js

```diff
@@ -0,0 +1,45 @@
+angular.module('privilegeService', ['ngResource'])
+    .config(function ($httpProvider) {
+        // to prevent the browser from displaying a password pop-up in case of an authentication error
+        $httpProvider.defaults.headers.common['Disable-WWW-Authenticate'] = 'true';
```

Comment:
I know we did this for encounterTypeService but i think it is not necessary because it is done globally for all pages [via this interceptor](https://github.com/openmrs/openmrs-module-uicommons/blob/master/omod/src/main/webapp/resources/scripts/angular-common.js#L15)